### PR TITLE
hd-streams.org

### PR DESCRIPTION
**Pull request by Venipa**
# hd-streams

### Viewing Movie/Series - Play, Pause & Browsing
image
image
image

(Series, same like movies except the view series button)
image

### Default
image

### CODE REVIEW

**Comment by Venipa** [comment]
#1400

File: websites/H/Hd-Streams/presence.ts [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+            searching: "general.search",
+            searchFor: "general.searchFor"
+          },
+          newLang
+        );
+      }
+      const presenceData: PresenceData = {
+          largeImageKey: IMAGES.LOGO,
+          largeImageText: app.metadata.service
+        } as PresenceData,
```
Comment (Bas950) [inline websites/H/Hd-Streams/presence.ts:220]: : PresenceData
as PresenceData

Shouldnt be needed double??

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by Bas950 — APPROVED** [review]
LGTM

**Review by ririxidev — APPROVED** [review]
lgtm 👍

**Pull request merged**